Card/src/main/java/com/paypal/android/card/Card.kt

```diff
@@ -23,5 +25,20 @@ data class Card(
     /**
      * The card's security code (CVV, CVC, CVN, CVE, or CID)
      */
-    var securityCode: String = ""
+    var securityCode: String? = null,
+
+    /**
+     * The card holder's name as it appears on the card
+     */
+    var name: String? = null,
```

Comment:
```suggestion
    var cardholderName: String? = null,
```

Name might be confused with something optional like a "nickname"